src/test/java/ui/RegisterTest.java

```diff
@@ -1,19 +1,20 @@
 package ui;
 
+import externalUIPages.WordpressPage;
 import org.testng.Assert;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
-import uiPages.BasePage;
 import uiPages.RegisterPage;
-import static org.testng.Assert.*;
-
-import static helpers.Utilities.getPropertyFromRegProp;
 
+import static helpers.Utilities.getPropertyFileContent;
+import static helpers.Utilities.getPropertyValue;
 
 
 public class RegisterTest extends BaseTest {
 
     private RegisterPage registerPage = null;
+    private WordpressPage wordpressPage = null;
+    public static final String REG_FILE = "./register.properties";
```

Comment:
Since the REG_FILE constant is and it will be used only in this class, then it should be private.

Reply:
Done